tools/refguide_check.py

```diff
@@ -854,6 +858,43 @@ def init_matplotlib():
         HAVE_MATPLOTLIB = False
 
 
+def check_dist_keyword_names():
+
+    distnames = set(distdata[0] for distdata in distcont + distdiscrete)
+    mod_results = []
+    for distname in distnames:
+        dist = getattr(stats, distname)
+        method_members = inspect.getmembers(dist, predicate=inspect.ismethod)
+        method_names = [method[0] for method in method_members
+                        if not method[0].startswith('_')]
+        for methodname in method_names:
+            dist = getattr(stats, distname)
+            method = getattr(dist, methodname)
+            try:
+                params = NumpyDocString(method.__doc__)['Parameters']
```

Comment:
I found this way of looking for the parameter before I found `inspect.signature`.

Reply:
I added parameter names found with `inspect.signature` to the set of parameter names. 
We should also check any discrepancies between these sets to see if there are undocumented parameters, but that is a separate issue.